cmd/daemon/start.go

```diff
@@ -72,3 +91,40 @@ func buildStartCmd(parentCmd *cobra.Command) {
 		select {}
 	}
 }
+
+// isAlreadyRunning checks if an instance of the application is already running.
+func isAlreadyRunning(pidFile string) bool {
+	if data, err := os.ReadFile(pidFile); err == nil {
+		pid, err := strconv.Atoi(string(data))
+		if err == nil && pidExists(pid) {
+			return true // PID found and process is running
+		}
+	}
+	return false
+}
+
+// pidExists checks if a given PID is currently active.
+func pidExists(pid int) bool {
+	if pid <= 0 {
+		return false
+	}
+
+	if runtime.GOOS == "windows" {
+		pidStr := strconv.Itoa(pid)
+		windowsCmd := exec.Command("tasklist", "/FI", "PID eq "+pidStr)
```

Comment:
```suggestion
		windowsCmd := exec.Command("tasklist", "/FI", "PID eq "+pidStr) //#nosec 204
```

you can fix this issue, with disabling this rule for this line only.

Reply:
Thank you for the this. 👍🏻